lib/WebService/Async/Onfido/RateLimiter.pm

```diff
@@ -0,0 +1,152 @@
+package WebService::Async::Onfido::RateLimiter;
```

Comment:
This file is copied from https://github.com/binary-com/perl-WebService-Async-CustomerIO/blob/master/lib/WebService/Async/CustomerIO/RateLimiter.pm